drivers/pinctrl/renesas/rcar/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+if (CONFIG_SOC_R8A77951 OR CONFIG_SOC_ARM64_R8A77951)
+  zephyr_library_sources(pfc_r8a77951.c)
+endif()
+
+zephyr_library_sources_ifdef(CONFIG_SOC_R8A77961 pfc_r8a77961.c)
+zephyr_library_sources_ifdef(CONFIG_SOC_R8A779F0 pfc_r8a779f0.c)
+
+zephyr_library_sources(pfc_rcar.c)
```

please move global source first

```suggestion
zephyr_library_sources(pfc_rcar.c)

if (CONFIG_SOC_R8A77951 OR CONFIG_SOC_ARM64_R8A77951)
  zephyr_library_sources(pfc_r8a77951.c)
endif()

zephyr_library_sources_ifdef(CONFIG_SOC_R8A77961 pfc_r8a77961.c)
zephyr_library_sources_ifdef(CONFIG_SOC_R8A779F0 pfc_r8a779f0.c)
```

---

I see no reason for doing this I just save the same building list as it was before socs files first and then files from driver dir

---

It was a nit-picks, trying to sort things.
Global -> specific

Thanks